entity-test-lib/src/main/java/org/terracotta/entity/PassThroughEntityActiveInvokeChannel.java

```diff
@@ -1,37 +1,34 @@
 package org.terracotta.entity;
 
 import org.terracotta.exception.EntityException;
-import org.terracotta.exception.EntityServerException;
 
 public class PassThroughEntityActiveInvokeChannel<R extends EntityResponse> implements ActiveInvokeChannel<R> {
   
-  private final InvokeMonitor<R> monitor;
   private volatile boolean closed = false;
 
-  public PassThroughEntityActiveInvokeChannel(InvokeMonitor<R> monitor) {
-    this.monitor = monitor;
+  public PassThroughEntityActiveInvokeChannel() {
   }
 
   @Override
   public synchronized void sendResponse(R r) {
     checkClosed();
-    monitor.accept(r);
+    throw new UnsupportedOperationException();
```

Comment:
It probably should... but now that I look at this stuff again, none of these classes seem to be being used?

Reply:
My quick digging around suggests nothing is using it (that can't be trivially removed as a dependency). Suggest I remove it as part of this PR?

Reply:
I'd leave it.  It is a better API than the deferment stuff currently being used.  Plus it is integrated with the API rather than the kludgy extension.  I'll look at migrating once this is done.  This is good enough for now.